Lecture19.md

```diff
@@ -0,0 +1,94 @@
+# Lecture Notes For 3/29/23 - Lecture 19
```

Comment:
As per style guide (#116 , https://github.com/kevinaangstadt/CS340-S23-Notes/pull/116/commits/11cf517f7ee2cc8450d988ac02cf9bdf709be625), we also need to change the name of this file to something like `Day19Notes.md`

Sorry to add this on, didn't notice this on my first read through!